envbuilder.go

```diff
@@ -83,68 +84,68 @@ type DockerConfig configfile.ConfigFile
 // Logger is the logf to use for all operations.
 // Filesystem is the filesystem to use for all operations.
 // Defaults to the host filesystem.
-func Run(ctx context.Context, options Options) error {
+func Run(ctx context.Context, opts options.Options) error {
```

Comment:
Can you justify why we need this change in the PR description? Not sure if having a package just for options isn't an antipattern in Go. It smells more like Java...

Reply:
When we import this from a separate repo I'd rather not have to import the whole of envbuilder and instead only the things we need.

Reply:
I guess this is opinionated, but this could land in `/pkg/api`.